eval/amt/1.amti_create_input_data.py

```diff
@@ -84,18 +87,112 @@ def process_single_file(start, end, max_count):
             fout.write(json.dumps(map) + "\n")
 
 
+def process_default_test_tasks(max_count):
```

Comment:
I think this is maxing over instances (not tasks); right?  if so, let's change the name so that it reflects it (e.g., `max_instance_count`).

Reply:
Updated.